big_tests/tests/mongoose_helper.erl

```diff
@@ -516,3 +517,17 @@ print_debug_info_for_module(Module) ->
     IqConfig = rpc(mim(), ets, tab2list, [sm_iqtable]),
     ct:pal("hosts_and_opts=~p~n iq_handlers=~p~n",
            [ModConfig, IqConfig]).
+
+get_vcard_config(Config) ->
```

Comment:
We could just use the config from `test.config` instead of using it.

Reply:
adjusted. we more or less do the same in some other backends.
It is just one option `backend` usually.